### ISSUE TITLE
Multi select dropdown gets error in DAP for multiple tabable elements

### ISSUE THREAD

**Issue by dmorgant** [carbon-design-system/carbon#3114]
## Environment

> Operating system - Any

> Browser - Any

> Automated testing tool and ruleset  - DAP latest ruleset (I think 1/2019)

> Assistive technology used to verify - DAP

## Detailed description
I implemented the multi-select dropdown as shown on the web page https://www.carbondesignsystem.com/components/dropdown/code#main-content

<img width="647" alt="image" src="https://user-images.githubusercontent.com/29490139/59777548-80700080-927a-11e9-971f-0684a43297e0.png">

I got an error in DAP that also shows up for the above mentioned web page

![image](https://user-images.githubusercontent.com/29490139/59777923-26236f80-927b-11e9-9dd4-472068155a34.png)

Which references this rule: 
https://aat.w3ibm.mybluemix.net/token/de8df1ce-98b3-4bb3-9c26-f1e43383ab4a/637a6695-357b-4877-948c-14a25de53d98/archives/2019FebDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1178.html

The example shown on the above mentioned web page is rather interesting in that keyboard access to the button with the title 'Clear all selected items' does not appear to be keyboard accessible..... or at least there is no focus indication set on the item when it is the active element.

For the 'X' image of this button, DAP is also showing: 
![image](https://user-images.githubusercontent.com/29490139/59779316-a5b23e00-927d-11e9-9074-d26e23a10262.png)

Which references this rule:
https://aat.w3ibm.mybluemix.net/token/de8df1ce-98b3-4bb3-9c26-f1e43383ab4a/637a6695-357b-4877-948c-14a25de53d98/archives/2019FebDeploy//doc/w3/help/en-US/idhi_accessibility_check_g1128.html

**Comment by emyarod** [comment]
if I'm not mistaken, this is related to #2918 and should be resolved in #2936

the examples on https://www.carbondesignsystem.com/ don't reflect the latest iteration of the components, which can be found on [Storybook](http://react.carbondesignsystem.com). Does the error exist for the `MultiSelect` storybook example?
